feature_gate/client.py

```diff
@@ -4,26 +4,35 @@ class FeatureNotFound(ValueError):
 class Client:
   def __init__(self, adapter):
     self.adapter = adapter
+    self._logger = adapter.logger
 
   def adapter(self):
     return self.adapter
 
+  def logger(self):
+    return self._logger
+
   def add(self, feature):
+    self.logger.info("add feature", feature=feature)
     return self.adapter.add(feature)
 
   def remove(self, feature):
+    self.logger.info("remove feature", feature=feature)
     return self.adapter.remove(feature)
 
   def features(self):
     return self.adapter.features()
 
   def is_enabled(self, feature):
+    self.logger.info("feature is_endabled", feature=feature)
```

Comment:
```suggestion
    self.logger.info("feature is_enabled", feature=feature)
```

Need to log the response (true/false) here too

Reply:
Typo fixed in [ce9dee5](https://github.com/deftinc/feature_gate/pull/14/commits/ce9dee5ca84ff852b73e958b2a4fec993651ec88).

I was fighting with testing even doing this simplest thing possible. Once some tests were passing in the easy case, the response logging was added in [0d59f04](https://github.com/deftinc/feature_gate/pull/14/commits/0d59f04198ad48d282f819f10b961ca72aef86de).